plugins/MUN Identity/__init__.py

```diff
@@ -111,3 +111,32 @@ async def identity_check(self, ctx: commands.Context, user: discord.Member):
             embed.colour = discord.Colour.red()
             embed.add_field(name="MUN Username", value="No username verified.")
             await ctx.send(embed=embed)
+
+    @identity.command(name="associate")
+    @commands.has_permissions(manage_messages=True)
```

Comment:
the rest of the administrative commands go off of audit log perms, any reason this is message perms?

```suggestion
    @commands.has_permissions(view_audit_log=True)
```

Reply:
It uses message permissions because viewing audit logs doesn't grant any permissions to modify anything, the check command just checks details that are visible in the audit log anyway. Where this modifies associations I wanted it to require more permissions